src/error.rs

```diff
@@ -41,7 +41,7 @@ pub enum Error {
         column: usize,
     },
 
-    #[doc(Hidden)]
+    #[doc(hidden)]

```

Comment:
But that's what I'm fixing. Is there a problem with the cache on the CI?

Reply:
Nope. `criterion` is being used as a dev-dependency, and it's importing this crate from crates.io.

Reply:
Ok, there is a cyclic dependency. 

But this is a criterion bug nightly so it should be fixed from the criterion repository. From here I can't do anything. Remove dependency, little more. 

It's just in the benchmarks I don't think it's important.